packages/clay-color-picker/src/index.tsx

```diff
@@ -511,17 +523,16 @@ const ClayColorPicker = ({
 											value.includes('var('))
 									) {
 										dispatch({
-											hex: color.toHex(),
+											hex: internalToHex(color),
 											hue: color.toHsv().h,
 										});
 
 										if (internalActive) {
 											const newColors = [...customColors];
 
-											newColors[state.splotch!] =
-												color.toHex();
-
-											onColorsChange(newColors);
+											(newColors[state.splotch!] =
+												internalToHex(color)),
+												onColorsChange(newColors);
```

Comment:
This formatting is a little strange, maybe just leaving a blank space between the calls is enough here to not create confusion in the reading.

Reply:
I removed the brackets and added a blank line. Thanks @matuzalemsteles ! :smiley: